package.json

```diff
@@ -556,6 +563,16 @@
           "when": "view == mavenProjects && viewItem == maven:dependenciesMenu",
           "group": "inline@0"
         },
+        {
+          "command": "maven.project.addFavorite",
+          "when": "view == javaProjectExplorer && viewItem == maven:favoritesMenu",
```

Comment:
I don't think we need this, as javaProjectExplorer is "Java Projects" view from [project manager extension](https://github.com/microsoft/vscode-java-dependency). And we don't contribute menu items there.